tools/docker/tests/test_flows.sh

```diff
@@ -137,8 +137,26 @@ test_client_association() {
 }
 
 test_higher_layer_data_payload_trigger() {
-    #TODO: Implement
-    return 1
+    status "test higher layer data payload"
+    
+    mac_gateway_hex=`echo $mac_gateway | tr --delete :`
+    mac_gateway_hex="0x${mac_gateway_hex}"
+    copies=200
+    
+    for i in `seq 1 $copies`
+    do
+        payload="$payload $mac_gateway_hex"
+    done
+
+    dbg "Send Higher Layer Data message"
+    eval send_bml_command '"bml_wfa_ca_controller \"DEV_SEND_1905,DestALid,$mac_agent1,MessageTypeValue,0x8018,tlv_type,0xA0,tlv_length,\
```

Comment:
It will be useful if you can add a comment which explains what is being sent here as 1905 message, describing the TLVs that are part of the command.